### DISCUSSION TITLE
Whats the best way to achieve a page type where the user can specify content on both the left and right of a page

### DISCUSSION THREAD

**Discussion by danjq88** [PiranhaCMS/piranha.core#1597] [Q&A]
Hi, first time posting but have been working with PiranhaCMS for the last 6months and really enjoy it! 

I have a requirment to be able to split certain pages of a website completely in two, with various content on each side.
Obvioulsy, I know that we can use the columns block to get rows of content which would usually be fine, however, I want to be able to make the left and right side effectivley block groups. 

I have tried to create a custom ColumnsGroup which would accept a custom GenericBlockGroup but I think its is restricted so didnt work. I have also come close by using regions, however I can only get it working with HTMLFields.

Does anyone have any tips on how I could possibly acheive this?

**Comment by tidyui** [comment]
We've hade this request before, i.e having the possibility to have several sections for blocks on a content type. At the moment this isn't possible, but I've added issue #1598 for it so we can investigate what it would mean to introduce this support.

Best regards

**Reply by danjq88** [reply]
Thank You. 

I do really like how you have managed to make it simple to use and integrate with, ive already written some add-ons such as contact and bookings modules that worked very nicely. Splitting the page is literally the only thing i've not been able to do so great work!
